app/models/loaned_book.rb

```diff
@@ -0,0 +1,3 @@
+class LoanedBook < ApplicationRecord
+  enum status: [:rending, :returned , :overdated]
```

Comment:
```suggestion
  enum status: {rending: 0, returned: 1 , overdated: 2}
```
dùng cú pháp hash theo vì array nhé